src/pages/permissions/SsoNotification.tsx

```diff
@@ -0,0 +1,51 @@
+import type { FC } from "react";
+import { useState } from "react";
+import { Notification } from "@canonical/react-components";
+
+const loadClosed = () => {
+  const saved = localStorage.getItem("ssoNotificationClosed");
+  return Boolean(saved);
+};
+
+const saveClosed = () => {
+  localStorage.setItem("ssoNotificationClosed", "yes");
+};
+
+interface Props {
+  openSsoModal: () => void;
+  hasOidc: boolean;
+}
+
+const SsoNotification: FC<Props> = ({ openSsoModal, hasOidc }: Props) => {
+  const [closed, setClosed] = useState(loadClosed());
+
+  if (closed || hasOidc) {
+    return null;
+  }
+
+  const handleClose = () => {
+    saveClosed();
+    setClosed(true);
+  };
+
+  return (
+    <Notification
+      severity="information"
+      title="Did you know?"
+      onDismiss={handleClose}
+      actions={[
+        {
+          label: "Show me how",
+          onClick: () => {
+            openSsoModal();
+          },
+        },
+      ]}
+    >
+      LXD can be configured to log in using a single-signon provider instead of
```

Comment:
```suggestion
      LXD can be configured to log in using a single-sign on provider instead of
```
I think this should be with a space. Wdyt?

Reply:
I am not sure, I just copy pasted what's on @piperdeck 's Figma. Gemini says "**single sign-on**"

Reply:
> single sign-on

That looks good to me.